schemas/telemetry/regrets-reporter-update/regrets-reporter-update.4.schema.json

```diff
@@ -0,0 +1,448 @@
+{
+  "$comment": "See https://docs.google.com/document/d/104x-QmODN7CQwvKVDlB4D2BJzJ0Q-BKnluW5EA7vX7Q/edit",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "properties": {
+    "data_deletion_request": {
+      "properties": {
+        "extension_installation_error_reporting_uuid": {
+          "type": "string"
+        },
+        "extension_installation_uuid": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "event_metadata": {
+      "properties": {
+        "browser_info": {
+          "properties": {
+            "build_id": {
+              "type": "string"
+            },
+            "name": {
+              "type": "string"
+            },
+            "vendor": {
+              "type": "string"
+            },
+            "version": {
+              "type": "string"
+            }
+          },
+          "type": "object"
+        },
+        "client_timestamp": {
+          "type": "string"
+        },
+        "event_uuid": {
+          "type": "string"
+        },
+        "extension_installation_uuid": {
+          "type": "string"
+        },
+        "extension_version": {
+          "type": "string"
+        },
+        "total_amount_of_regret_reports": {
+          "type": "number"
+        }
+      },
+      "type": "object"
+    },
+    "regret_report": {
+      "properties": {
+        "form_step": {
+          "type": "number"
+        },
+        "report_data": {
+          "properties": {
+            "parent_youtube_navigations_metadata": {
+              "items": {
+                "description": "Selected/derived attributes from YouTubeNavigations",
+                "properties": {
+                  "document_visible_time": {
+                    "type": "number"
+                  },
+                  "page_entry_point": {},
```

Comment:
The generated schema is missing the page entry point.

Reply:
Thanks @acmiyaguchi I missed this during a recent update, and as @mreid-moz suggested offline, a file was improperly named. This has now been addressed.